.gitignore

```diff
@@ -16,7 +16,6 @@ Thumbs.db
 
 # Wordpress
 wp-content/*
-!wp-content/themes/
```

Comment:
The idea of this was to whitelist the themes you want. Is there a reason you removed it?